stix_shifter_modules/splunk/stix_translation/query_constructor.py

```diff
@@ -190,9 +187,32 @@ def translate(self, expression):
                 self.translate(expression.expr2)
             )
 
+    def _field_severity(self, value):
+        """
+        check for severity and convert input value to
+        informational,low,medium,high,critical
+        param value
+        return value(str)
+        """
+        value = int(value)
+        if 1 <= value <= 20:
+            value = str("informational")
```

Comment:
`"informational"` etc. are already `str`s

Reply:
removed the str()

Reply:
I still see them.  Did you forget to push?

Reply:
No. The changes are there, please see below commit reference.

https://github.com/opencybersecurityalliance/stix-shifter/pull/1479/commits/0d09723a07357f899962fd3cc2e14c67976d01c2